src/utils/bbox.ts

```diff
@@ -7,9 +7,17 @@ export function getBoundingBoxByChildren(context: Context, svgnode: SvgNode): nu
   if (getAttribute(svgnode.element, context.styleSheets, 'display') === 'none') {
     return [0, 0, 0, 0]
   }
-  let boundingBox = [0, 0, 0, 0]
+  let boundingBox : number[] = [];
   svgnode.children.forEach(child => {
     const nodeBox = child.getBoundingBox(context)
+    if ((nodeBox[0] === 0) && (nodeBox[1] === 0) && (nodeBox[2] === 0) && (nodeBox[3] === 0))
+       return;
+    const transform = child.computeNodeTransform(context)
+    nodeBox[0] = nodeBox[0] * transform.sx + transform.tx
+    nodeBox[1] = nodeBox[1] * transform.sy + transform.ty
+    if (boundingBox.length === 0)
+      boundingBox = nodeBox;
+    else
```

Comment:
The width and height of the bounding box also need to be transformed. You can use the multVecMatrix function from geometry.ts